src/main/scala/roguelike/model/SceneTime.scala

```diff
@@ -0,0 +1,5 @@
+package roguelike.model
+
+import indigo._
```

Comment:
In Scala 3, prefer: `import indigo.*` and `_` imports have been replaced with `*` to reduce ambiguity.